src/ui/profile_preferences.ui

```diff
@@ -2789,15 +2789,15 @@
        </item>
       </layout>
      </widget>
-     <widget class="QWidget" name="tab_security">
+     <widget class="QWidget" name="tab_connection">
       <attribute name="title">
-       <string>Security</string>
+       <string>Connection</string>
       </attribute>
-      <layout class="QGridLayout" name="gridLayout_tab_security">
-       <item row="0" column="0">
+      <layout class="QVBoxLayout" name="verticalLayout_3">
```

Comment:
Mind changing this item to be `verticalLayout_tab_connection` ? It helps with revisions if the layout instances are name with something derived from the widget they are applied to (instead of the default `xxxxLayout_` + numeric suffix)...